therapy/database.py

```diff
@@ -1,28 +1,36 @@
 """This module creates the database."""
 import boto3
 
-DYNAMODB = boto3.resource('dynamodb', endpoint_url="http://localhost:8000")
-DYNAMODBCLIENT = \
-    boto3.client('dynamodb', endpoint_url="http://localhost:8000")
-THERAPIES_TABLE = DYNAMODB.Table('Therapies')
-METADATA_TABLE = DYNAMODB.Table('Metadata')
-cached_sources = dict()
-
 
 class Database:
     """The database class."""
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, db_url='', region_name='us-east-2',
+                 create_tables=False):
         """Initialize Database class."""
-        existing_tables = DYNAMODBCLIENT.list_tables()['TableNames']
-        self.create_therapies_table(DYNAMODB, existing_tables)
-        self.create_meta_data_table(DYNAMODB, existing_tables)
+        if db_url:
+            self.ddb = boto3.resource('dynamodb', region_name=region_name,
+                                      endpoint_url=db_url)
+            self.ddb_client = boto3.resource('dynamodb',
+                                             region_name=region_name,
+                                             endpoint_url=db_url)
```

Comment:
This never gets called. We should add a flag in cli.py to determine if we are in dev/prod